app/tests/fixtures/credentials.py

```diff
@@ -433,3 +433,79 @@ async def get_or_issue_regression_cred_valid(
         referent=valid_credential["referent"],
         cred_def_revocable=valid_credential["cred_def_id"],
     )
+
+
+@pytest.fixture(scope="function")
+async def issue_alice_creds_non_revoke(
```

Comment:
Test should probably be skipped in regression mode, to prevent bloating of regression fixtures.